Extension/test/integrationTests/testAssets/SingleRootProject/references.cpp

```diff
@@ -0,0 +1,54 @@
+namespace foo {
+	namespace bar {
+		namespace baz {
+			int qux = 42;
+		}
+	}
+}

```

Comment:
I got this test sample from https://en.cppreference.com/w/cpp/language/namespace_alias -- looks like there's some copyright restrictions -- maybe we should change the test case? https://en.cppreference.com/w/Cppreference:Copyright/CC-BY-SA

Reply:
Moving these test assets into an internal repro so we don't to publish it.